src/views/Home.vue

```diff
@@ -2,6 +2,20 @@
   <h2 class="text-h6 pb-4">
     {{ $t('Common.TaglineAlt1') }} {{ APP_HOME_ICONS }}
   </h2>
+  <v-row>
```

Comment:
on mobile, it takes half of the page

<img width="425" height="888" alt="Image" src="https://github.com/user-attachments/assets/2c2d03ca-07b7-47d5-8777-ace35df5596f" />